guides/extensions/embedded-scripts/embedded-script-task.md

```diff
@@ -80,151 +53,40 @@ Your file structure should look like this:
-    ```
-
-2. Make sure this file includes an object containing key-value pairs for each of your dynamic parameters.
-
-   For example, for the code:
-
-    ```tsx
-    <script> console.log("Hello {{userName}} from the CLI. Your Google Tag ID is: {{googleTagId}}."); </script>
-    ```
-
-    Requires a params.dev.json file in the following format:
-
-    ```tsx
-    {
-      "userName": "Jerry",
-      "googleTagId": "GT-XXXXXXXXX"
-    }
-    ```
-
-Make sure the keys are the dynamic parameter names in quotes. The values will be assigned to the parameters when testing your script.
-
-## Add global CSS to your HTML code (Optional)
-
-You can add CSS directly to your `embedded.html` file, or you can reference a CSS stylesheet with a link. For example:
-
-  ```tsx
-  <link rel="stylesheet" href="./<your-css-file>.css"/>
-  ```
-
-This CSS applies to your site globally. For example, the following code would make the background of every page of your site red:
-
-  ```tsx
-  #c1dmp {
-    background: red;
-  }
-  ```
-
-## Prepare your app for production
+## Step 2 | Prepare your app for production
 
 To finish setting up your embedded script, either you or the site owner must call the [Embed Script](https://dev.wix.com/docs/rest/api-reference/app-management/apps/embedded-scripts/embed-script) endpoint to embed your script and update the values of the dynamic parameters in each app instance.
 
-### Prompting the site owner to embed the code (recommended)
+#### Prompting the site owner to embed the code (recommended)
 
 If your app has a dashboard page, you have the option to shift responsibility for this last configuration step onto site owners.
 
-Site owners can call the [Embed Script](https://dev.wix.com/docs/rest/api-reference/app-management/apps/embedded-scripts/embed-script) endpoint using the `fetch` method from the [Wix React SDK](https://dev.wix.com/docs/sdk/api-reference/sdk-react/setup).
-
 >**Note**: If an app has a dashboard page and an embedded script extension, site owners will automatically be directed to the app's dashboard page after installing the app.
 
-This API call is also used to specify the value of any dynamic parameters. For more information about using dynamic parameters see (See [Using dynamic parameters in your HTML code](#using-dynamic-parameters-in-your-html-code-optional)).
+Site owners can call the [Embed Script](https://dev.wix.com/docs/rest/api-reference/app-management/apps/embedded-scripts/embed-script) endpoint using the `fetch` method from the [Wix React SDK](https://dev.wix.com/docs/sdk/api-reference/sdk-react/setup).
+
+This API call is also used to specify the value of any dynamic parameters. For more information about using dynamic parameters see [Using dynamic parameters in your HTML code](./embedded-script-extension-files-and-code.md#using-dynamic-parameters-in-your-html-code).
 
 To use the `fetch` method in your app's dashboard page:
 
-1. Navigate to your `page.tsx` file in `src/dashboard/pages`.
```

Comment:
@dor-chaouat correct me if I'm wrong but this can't be done before the user run `create-version`

Reply:
Added a note that this is just setup and you can't actually make the call until you create a version.